node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+
+const data = (Math.floor(Math.random() * Math.floor(100))).toString();
+
+fs.writeFile('./random-txt', data, 'utf8', err => {
+  if (err) {
```

Comment:
It's a good idea to `console.error()` the error as well, so that the program doesn't just fail silently.